apps/src/aichat/types.ts

```diff
@@ -5,41 +5,44 @@ import {Role} from '../aiComponentLibrary/chatItems/types';
 // TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
 export type AichatInteractionStatusValue = string;
 
-export interface ChatItem {
+export interface AichatEvent {
   // UTC timestamp in milliseconds
   timestamp: number;
+  description?: string;
 }
 
-export interface ChatMessage extends ChatItem {
+export interface ChatMessage extends AichatEvent {
   chatMessageText: string;
   role: Role;
   status: AichatInteractionStatusValue;
 }
 
-export interface ModelUpdate extends ChatItem {
+export interface ModelUpdate extends AichatEvent {
   id: number;
   updatedField: keyof AiCustomizations;
   updatedValue: AiCustomizations[keyof AiCustomizations];
 }
 
-export interface Notification extends ChatItem {
+export interface Notification extends AichatEvent {
   id: number;
   text: string;
   notificationType: 'error' | 'success';
 }
 
-// Type Predicates: checks if a ChatItem is a given type, and more helpfully,
+export type ChatItem = ChatMessage | ModelUpdate | Notification;
```

Comment:
Can we add some comments to explain the differences between `ChatItem`s and `AichatEvent`s?

Reply:
How does this sound:
```
// ChatItems are types of messages that are displayed for the user in the chat workspace.
// ChatEvents that are not ChatItems are NOT displayed to the user. However, they
// are displayed when a teacher views a student's chat history in the chat workspace.
// An example of a ChatEvent that is not a ChatItem is 'The user cleared the chat workspace.'
```